calendar-service/calendar-service.sh

```diff
@@ -33,25 +33,24 @@ start() {
 
   if [ "$pid" -ne 0 ] ; then
     echo "Calendar service is already started"
-    exit 1
+  else
+    # If the application isn't running, starts it
+    echo " *** Starting Calendar Service *** "
+
+    #copy scheduler users to radicale server
+    loginFile="$cwd/config/authentication/login.cfg"
+    rm -rf ~/.config/radicale/users
+    touch ~/.config/radicale/users
+
+    while IFS=':' read -r user password
+    do
+        htpasswd -bs ~/.config/radicale/users "$user" "$password"    
+    done < "$loginFile"
+
+    # Redirects default and error output to a log file
+    java -Dpa.scheduler.home="$cwd" -Dspring.config.location="$cwd/config/calendar-service/application.properties" -jar "$cwd"/calendar-service/calendar-service*.jar > /dev/null 2>&1 &
```

Comment:
Since the calendar service is not started by the Scheduler Jetty server, why is the calendar-service configuration file in scheduler _config_ folder? is it because you need to have access to some of its properties from scheduler portal and rm portal?